app/views/foods/general.html.erb

```diff
@@ -0,0 +1,26 @@
+
+<div class="List">
+<h1 class="heading">Shopping List</h1>
+<div class="shopping-info">
+  <div><span class="">Amount of food items to buy: </span><span class="">&nbsp;<%= @foods.length %></span></div>
+    <div><span class="">Total value of food needed:</span><span class="">&nbsp;$<%= @total %></span></div>
+</div>
+<table class="">
```

Comment:
[OPTIONAL] This is only optional because it was not mentioned by the previous reviewer, but I highly recommend that you take it into consideration because it is part of the project requirements.

The `Shopping list` page does not display any ingredients although the Recipe has a few ingredients that I added.

![image](https://user-images.githubusercontent.com/38088818/218204061-01d25135-6159-4d58-9c4a-36eb2564322c.png)

The Recipe:

![image](https://user-images.githubusercontent.com/38088818/218204148-f5fa4b35-c8fa-4377-b8c1-e1be86c965da.png)